examples/solenoid_softedge/run_solenoid_softedge.py

```diff
@@ -124,7 +124,16 @@
     nslice=4,
 )
 
-sim.lattice.extend([sol])
+# add beam diagnostics
+monitor = elements.BeamMonitor("monitor", "h5")
+
+sim.lattice.extend(
+    [
+        monitor,
+        sol,
+        monitor,
+    ]
+)
 
 # run simulation
 sim.evolve()
```

Comment:
Kudos for making it through a very tedious set of changes.

Reply:
Thanks, haha :sweat_smile: